features/support/sphinx.rb

```diff
@@ -10,11 +10,19 @@
   ::ThinkingSphinx::Test.clear
   ::ThinkingSphinx::Test.init
   ::ThinkingSphinx::Test.stop
-  ::ThinkingSphinx::Test.autostop
+  $searchd_autostop_installed ||= ::ThinkingSphinx::Test.autostop
```

Comment:
Where does `$searchd_autostop_installed` come from?

Reply:
This is just to prevent registering numerous `at_exit` hooks. Wait with this PR. It's not ready.

Reply:
why the `$` prefix?

Reply:
global variable, do you have other suggestions?

Reply:
Not really, it just seems weird to start a variable with `$`. It looks like Javascript. Is that a convention?

Reply:
It is syntax. Like `@`  or `@@`.

Reply:
Thanks. I should learn ruby one of these days.